schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT PRIMARY KEY NOT NULL,
+    name VARCHAR(100) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INT NOT NULL,
+    neutered BOOLEAN NOT NULL, 
+    weight_kg DECIMAL NOT NULL,
 );
```

Comment:
- [Optional] Dear @Tamunokuro, Good job creating the table. but I suggest you make the id auto increment IDENTITY(1,1). 

```sql
id int NOT NULL IDENTITY(1,1)
```